### ISSUE TITLE
mpl_set_parsim_t(1, tname="FITCH_T", morphy)

### ISSUE THREAD

**Issue by ms609** [mbrazeau/MorphyLib#5]
`mpl_set_parsim_t(1, tname="FITCH_T", morphy)` returns error code 12, "Character type is unknown."

I've tried `tname="FITCH_T"` and `tname = 1` but receive the same error.

**Comment by ms609** [comment]
With some trial and error, `tname="FITCH"` works.  Perhaps the documentation could be updated with a clearer list of acceptable values?

**Comment by mbrazeau** [comment]
"FITCH_T" is an enumerated C type. Just use "fitch" or "FITCH" or "Fitch".

**Comment by mbrazeau** [comment]
Yes. The R documentation needs to be updated with the particular usage for the R functions. Rather than re-defining the enumerated types in R, I've just written a parser for verbose names.
